Content.Shared/GameObjects/Components/Mobs/State/SharedCriticalMobState.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Content.Shared.Alert;
+using Content.Shared.GameObjects.EntitySystems;
+using Robust.Shared.GameObjects.Systems;
+using Robust.Shared.Interfaces.GameObjects;
+
+namespace Content.Shared.GameObjects.Components.Mobs.State
+{
+    /// <summary>
+    ///     A state in which an entity is disabled from acting due to sufficient damage (considered unconscious).
+    /// </summary>
+    public abstract class SharedCriticalMobState : BaseMobState
+    {
+        protected override DamageState DamageState => DamageState.Critical;
+
+        public override void EnterState(IEntity entity)
+        {
+            base.EnterState(entity);
+
+            if (entity.TryGetComponent(out SharedAlertsComponent status))
+            {
+                status.ShowAlert(AlertType.HumanCrit); // TODO: combine humancrit-0 and humancrit-1 into a gif and display it
+            }
+        }
+
+        public override void ExitState(IEntity entity)
+        {
+            base.ExitState(entity);
+
+            EntitySystem.Get<SharedStandingStateSystem>().Standing(entity);
```

Comment:
Shouldn't this be an IActionBlocker?

Reply:
Additionally this the Content.Shared version of the component, so on further inspection you should remove this line entirely and have it be an IActionBlocker on the Content.Server side.

Reply:
Not change so much as just adding something, a way to know when blocking states change. It shouldn't be a problem for anything being developed now or require any coordination with me.

Reply:
How would this be turned into an IActionBlocker?